include/d/d_com_inf_game.h

```diff
@@ -378,15 +378,15 @@ inline void dComIfGs_onSaveTbox(int i_stageNo, int i_no) {
     g_dComIfG_gameInfo.save.getSavedata().getSave(i_stageNo).getBit().onTbox(i_no);
 }
 
-inline BOOL dComIfGs_isStageBossEnemy() {
+BOOL dComIfGs_isStageBossEnemy(int i_stageNo) {
```

Comment:
are you sure these are right? the parameter isnt getting used at all

Reply:
Ghidra's interpretation of this function is much more complex than the one that is in the repo. I assume that they simplified the function for TP?

Reply:
I didn't update the function's C++ because I was focusing on d_a_bossitem.

Reply:
there's two versions of the function according to tww debug map, one with an int param and one with void. maybe it wants the int one from the cpp file instead of this?

Reply:
Ahhh, I see! That's why it has the `extern`s. One second.